README.md

```diff
@@ -48,14 +48,22 @@ git clone https://github.com/zavolanlab/zarp.git
 cd zarp
 ```
 
-## 2. Conda installation
+## 2. Conda and Mamba installation
 
 Workflow dependencies can be conveniently installed with the [Conda][conda]
 package manager. We recommend that you install [Miniconda][miniconda-installation] 
 for your system (Linux). Be sure to select Python 3 option. 
 The workflow was built and tested with `miniconda 4.7.12`.
 Other versions are not guaranteed to work as expected.
 
+Given that Miniconda has been installed and is available in the current shell the first
+dependency for ZARP is the [Mamba][mamba] package manager, which needs to be installed in
+the `base` conda environment with:
+
+```bash
+conda install mamba -n base -c conda-forge
```

Comment:
That's good that you it mention here. I think that we should apply the same methodology in the tests. If we switch to mamba we will speed up a lot the testing process.

Reply:
Now, upon the new push - yes, docs updated ;)

Reply:
Update: `mamba env update` would not work, reverted that change, see https://github.com/mamba-org/mamba/issues/631